### PR TITLE
LUN-122: Add Imprint

### PR DESCRIPTION
**Pull request by Taggotty**
This pull request belongs to an issue on our [bugtracker](https://issues.tuerantuer.org/secure/RapidBoard.jspa?rapidView=27&view=detail).
You can find it there by looking for an issue with the key which is mentioned in the title of this pull request.
It starts with the keyword _LUN_.

### CODE REVIEW

File: src/components/Link.tsx [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+import styled from 'styled-components/native'
+
+import { openExternalUrl } from '../services/url'
+
+const LinkText = styled.Text`
+  text-decoration: underline;
+  font-size: ${props => props.theme.fonts.defaultFontSize};
+  font-family: ${props => props.theme.fonts.contentFontRegular};
+  font-weight: ${props => props.theme.fonts.lightFontWeight};
+  color: blue;
```
Comment (SteffiStuffel) [inline src/components/Link.tsx:12]: Would be nice if you can move this to `theme.colors`

File: src/components/HomeFooter.tsx [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+`
+
+interface HomeFooterProps {
+  navigateToImprint: () => void
+}
+
+const HomeFooter = ({ navigateToImprint }: HomeFooterProps): ReactElement => (
+  <Container>
+    <ContentSecondary>{'\u00A9'}LUNES2020</ContentSecondary>
+    <LinkSecondary onPress={navigateToImprint}>Impressum</LinkSecondary>
```
Comment (SteffiStuffel) [inline src/components/HomeFooter.tsx:34]: Can you please move "Impressum" to constant/labels.json

File: src/components/HomeFooter.tsx [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+  color: ${props => props.theme.colors.textSecondary};
+`
+
+interface HomeFooterProps {
+  navigateToImprint: () => void
+}
+
+const HomeFooter = ({ navigateToImprint }: HomeFooterProps): ReactElement => (
+  <Container>
+    <ContentSecondary>{'\u00A9'}LUNES2020</ContentSecondary>
```
Comment (klinzo) [inline src/components/HomeFooter.tsx:33]: Why 2020?

File: src/components/Link.tsx [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+`
+
+const Touchable = Platform.select<React.ComponentType<TouchableHighlightProps>>({
```
Comment (klinzo) [inline src/components/Link.tsx:15]: Why don't you just use a [pressable](https://reactnative.dev/docs/pressable)?

File: src/routes/ImprintScreen.tsx [hunk 5]
```diff
@@ -0,0 +1,75 @@
...
+import Link from '../components/Link'
+
+const Root = styled.View`
+  background-color: ${prop => prop.theme.colors.background};
+  height: 100%;
+  width: 100%;
+`
+
+const Container = styled.View`
+  margin: ${props => props.theme.spacings.sm};
```
Comment (SteffiStuffel) [inline src/routes/ImprintScreen.tsx:13]: You can just remove Container, if adding this margings as paddings in Root.

File: src/routes/ImprintScreen.tsx [hunk 6]
```diff
@@ -0,0 +1,75 @@
...
+`
+
+const Description = styled.Text`
```
Comment (SteffiStuffel) [inline src/routes/ImprintScreen.tsx:16]: If Andys PR (LUN-247) gets merged first, would be nice if you can use the text-components he created there.

**Review by klinzo — COMMENTED** [review]
I think the touch target is a little bit small

File: src/routes/ImprintScreen.tsx [hunk 7]
```diff
@@ -0,0 +1,75 @@
...
+  font-size: ${props => props.theme.fonts.defaultFontSize};
+  font-family: ${props => props.theme.fonts.contentFontRegular};
+  font-weight: ${props => props.theme.fonts.lightFontWeight};
+  color: ${props => props.theme.colors.text};
+  margin-bottom: ${props => props.theme.spacings.xs};
+`
+
+const Title = styled.Text`
+  text-align: left;
```
Comment (SteffiStuffel) [inline src/routes/ImprintScreen.tsx:25]: `left` is default anyways, so can be removed, same for other occurences.

File: src/routes/ImprintScreen.tsx [hunk 8]
```diff
@@ -0,0 +1,75 @@
...
+      <Description>Finanzamt Augsburg-Stadt</Description>
+      <Subheading>Sitz der Gesellschaft:</Subheading>
+      <Description>Augsburg</Description>
+      <Subheading>Handelsregister und Registernummer:</Subheading>
+      <Description>HRB30759</Description>
+      <Subheading>Umsatzsteuer-Identifikationsnummer:</Subheading>
+      <Description>DE307491397</Description>
+      <Subheading>Kontakt</Subheading>
+      <MultiTextContainer>
+        <Description>Telefon:</Description>
```
Comment (SteffiStuffel) [inline src/routes/ImprintScreen.tsx:64]: Missing space after colon.

**Review by SteffiStuffel — APPROVED** [review]
Tested on android. Looks good :tada:

**Comment by SteffiStuffel** [comment]
maybe you can add release notes for this. With `yarn create-realease-note "Impressum hinzugefügt"` it is really fast.

File: src/routes/ImprintScreen.tsx [hunk 9]
```diff
@@ -0,0 +1,75 @@
...
+  text-align: left;
+  font-size: ${props => props.theme.fonts.headingFontSize};
+  font-weight: ${props => props.theme.fonts.defaultFontWeight};
+  letter-spacing: ${props => props.theme.fonts.listTitleLetterSpacing};
+  margin: ${props => props.theme.spacings.sm} 0;
+  font-family: ${props => props.theme.fonts.contentFontBold};
+  color: ${props => props.theme.colors.text};
+`
+
+const Subheading = styled.Text`
```
Comment (f1sh1918) [inline src/routes/ImprintScreen.tsx:34]: you can now use `Subheading` component

File: src/routes/ImprintScreen.tsx [hunk 10]
```diff
@@ -0,0 +1,75 @@
...
+`
+
+const ImprintScreen = (): ReactElement => (
+  <Root>
+    <Container>
+      <Title>Impressum</Title>
+      <Description>Angaben gemäß § 5 TMG und verantwortlich für den Inhalt nach § 55 Abs. 2 RStV:</Description>
+      <Description>Tür an Tür – Digitalfabrik gGmbH Wertachstr. 29 86153 Augsburg</Description>
+      <Description>vertreten durch Herrn Daniel Kehne, Herrn Fritjof Knier</Description>
+      <Subheading>Aufsichtsbehörde</Subheading>
```
Comment (f1sh1918) [inline src/routes/ImprintScreen.tsx:54]:
```suggestion
      <Subheading>Aufsichtsbehörde: </Subheading>
```
```suggestion
      <Subheading>Aufsichtsbehörde</Subheading>
```

Comment (f1sh1918) [inline src/routes/ImprintScreen.tsx:54]:
```suggestion
      <Subheading>Aufsichtsbehörde: </Subheading>
```

**Review by f1sh1918 — APPROVED** [review]
Tested on ios

File: src/components/HomeFooter.tsx (hunk 3 above)
Comment (Taggotty) [reply src/components/HomeFooter.tsx:33]: just copied from the designs. Will change this to 2022

**Pull request merged**